### PR TITLE
Redesigned Search and Stocks You Follow

### PR DESCRIPTION
**Pull request by javidangarcia**

### CODE REVIEW

File: frontend/src/components/Search/Search.jsx [hunk 1]
```diff
@@ -25,12 +25,12 @@ export default function Search() {
...
                     />
                     <button type="submit">
-                        <i className="material-icons">search</i>
+                        <i className="fa fa-search"></i>
                     </button>
                 </form>
                 <SearchQuery searchInput={searchInput} />
             </div>
-            <SearchTable />
+            <StockCarousel />
```
Comment (vandanasub) [inline frontend/src/components/Search/Search.jsx:33]: good example of how having a component serve a single purpose makes it simpler to swap/replace, like the change you've made in this commit.

File: frontend/src/components/StocksYouFollow/StocksYouFollow.css [hunk 2]
```diff
@@ -12,43 +12,11 @@
     margin-bottom: 10px;
 }
 
-.stocks-you-follow table {
-    overflow: hidden;
-    width: 100%;
-}
-
```
Comment (vandanasub) [inline frontend/src/components/StocksYouFollow/StocksYouFollow.css:19]: nice, good to see simplified / deleted css

File: frontend/src/components/StocksYouFollow/StocksYouFollow.jsx [hunk 3]
```diff
@@ -1,79 +1,43 @@
...
-                                        {stock.ticker}
-                                    </Link>
-                                </td>
-                                <td className="text-center">${stock.price}</td>
-                                <td className="text-center">{stock.sector}</td>
-                            </tr>
-                        ))}
-                </tbody>
-            </Table>
+            {stocks
```
Comment (vandanasub) [inline frontend/src/components/StocksYouFollow/StocksYouFollow.jsx:15]: Can you share what your design thoughts were on the move from table to cards?

**Review by vandanasub — COMMENTED** [review]
Looks great!

File: frontend/src/components/StocksYouFollow/StocksYouFollow.jsx (hunk 3 above)
Comment (javidangarcia) [reply frontend/src/components/StocksYouFollow/StocksYouFollow.jsx:15]: The cards look much better than the table and it allows users to click anywhere on the card to navigate to the stock page.

**Pull request merged**